tutorials/pub-sub/README.md

```diff
@@ -331,7 +331,55 @@ dapr stop --app-id react-form
 
 <!-- END_STEP -->
 
-4. If you want to deploy this same application to Kubernetes, move onto the next step. Otherwise, skip ahead to the [How it Works](#How-it-Works) section to understand the code!
+4. If you want more insights into the cloud events being published on Redis streams, move onto the next section. If you want to deploy this same application to Kubernetes, move onto the section after that. Otherwise, skip ahead to the [How it Works](#How-it-Works) section to understand the code!
+
+## Insights into the cloud events being published on Redis streams
+If you want to see the [Cloud Events](https://github.com/cloudevents/spec) that are being published on [Redis stream](https://redis.io/docs/data-types/streams/), follow the below steps: 
+
+1. Redis is running inside of a docker container, so to access it via CLI, we pass the following command to a new terminal:
```

Comment:
```suggestion
1. Redis is running inside of a docker container, so to access it via CLI, pass the following command to a new terminal:
```

Reply:
@shivanisinghnitp - Can you review these suggestions? Thanks

Reply:
@shivanisinghnitp - Pinging again

Reply:
@msfussell sure will review the changes